### ISSUE TITLE
Subsystem operation problem ? 

### ISSUE THREAD

**Issue by WuZihao12** [BingqiShen/EMV-LIO#2]
Hi, thanks for the great work, can your proposed system run multi-camera subsystem or lio subsystem alone ?

**Comment by BingqiShen** [comment]
YEAH, it can work alone on both subsystem and here is the screencast when vio subsystem works alone
![微信图片_20231215163340](https://github.com/BingqiShen/EMV-LIO/assets/57284869/93eafdd2-a65e-4ddd-a780-3ac0539c4bff
![微信图片_20231215163340](https://github.com/BingqiShen/EMV-LIO/assets/57284869/7ef026aa-1e84-4e96-aa2c-132f7d093a49)

**Comment by WuZihao12** [comment]
Thank you for your reply. Can you send me the parameter configuration file and launch file of Newer College Multi-Camera Dataset?

**Comment by BingqiShen** [comment]
> Thank you for your reply. Can you send me the parameter configuration file and launch file of Newer College Multi-Camera Dataset?

[module_sam_oxford.zip](https://github.com/BingqiShen/EMV-LIO/files/13971355/module_sam_oxford.zip)
You can try this configuration file~
